ipv8/src/main/java/nl/tudelft/ipv8/attestation/trustchain/validation/TransactionValidator.kt

```diff
@@ -4,5 +4,5 @@ import nl.tudelft.ipv8.attestation.trustchain.TrustChainBlock
 import nl.tudelft.ipv8.attestation.trustchain.store.TrustChainStore
 
 interface TransactionValidator {
-    fun validate(block: TrustChainBlock, database: TrustChainStore): Boolean
```

Comment:
I'm getting errors through this change when running a trustchain-superapp build on 15/02

Reply:
![image](https://user-images.githubusercontent.com/4957928/107990600-08208c00-6fd5-11eb-8bed-600683061d32.png)

Reply:
I already fixed this. Are you up to date with the newest commit of the super app?

Reply:
If this is new code: Replace True with ValidationResult.Valid, and False with ValidationResult.Invalid.